### ISSUE TITLE
Scrollbar isn't inside container div it's fixed to the bottom of the page

### ISSUE THREAD

**Issue by richaus** [imanhodjaev/ember-perfect-scroll#28]
No matter what container I wrap with {{#perfect-scroll}} the scrollbar (_ps__rail-x_ and  _ps__rail-y_divs) ends up fixed to the bottom of the page. It appears to be the correct width... just absolute positioned at the bottom (left) of the page.

**Comment by richaus** [comment]
Added an additional div outside of with  {{#perfect-scroll}} css style _position:relative_ and that appears to work more or less. Guess this will just require more css tinkering. Thanks!

**Comment by richaus** [comment]
Just wanted to add to what is happening. So now the scrollbar extends over the content (albeit in the correct place now) but as you scroll it causes an additional scrollbar to show up on the page (not perfect-scroll styled)... so I added _overflow:hidden_ to the same div as I added _position:relative_ to and that seems to have worked.

Just adding all this so it maybe will help someone down the road. Good luck folks!
